lib/internal/test_runner/reporter/dot.js

```diff
@@ -3,19 +3,19 @@ const {
   ArrayPrototypePush,
   MathMax,
 } = primordials;
-const colors = require('internal/util/colors');
 const { formatTestReport } = require('internal/test_runner/reporter/utils');
+const colors = require('internal/util/colors');
```

Comment:
This seems like an unrelated change, could you revert it?

Reply:
@atlowChemi done!